e2e-tests/cypress/tests/integration/channels/toast/new_messages_toast_spec.js

```diff
@@ -138,7 +138,7 @@ describe('Toast', () => {
             cy.uiClickSidebarItem(testChannelName);
 
             // # Scroll up so bottom is not visible
-            scrollUp();
+            scrollUpAndPostAMessage(otherUser, testChannelId, oldPostNumber);
 
             // # New message toast is present with expected number of messages
             cy.get('div.toast').should('be.visible');
```

Comment:
MM-T1786 failed

Reply:
this passes locally, earlier it used to fail in local as well.
<img width="526" alt="image" src="https://github.com/mattermost/mattermost/assets/15979783/7bb5b1b1-ff47-41fe-a014-aee81831c31a">